src/Aqovia.DurableFunctions.Testing/Aqovia.DurableFunctions.Testing.csproj

```diff
@@ -23,8 +23,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Azure.ServiceBus" Version="4.1.2" />
-		<PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.18" />
-		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.5.0" />
+		<PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.31" />
+		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.8.1" />
```

Comment:
Required to give us access to `IOrchestrationServiceQueryClient.GetOrchestrationWithQueryAsync` which we can use to fetch all orchestrators or fetch by query, for situations where we don't know the instance Id.